tools/gnur-runtime/src/main/java/org/renjin/gnur/Sort.java

```diff
@@ -33,6 +35,35 @@
    * larger to right
    *
    */
+  public static void iPsort2(IntPtr x, int lo, int hi, int k) {
```

Comment:
All pointer arguments should now accept the `Ptr` interface rather than any of the implementation classes. This allows us to support C code that casts pointers to different primitive types.